<?php

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

// This file has been auto-generated by the Symfony Dependency Injection Component for internal use.
// Returns the private '.service_locator.Yva8Kfp' shared service.

return $this->privates['.service_locator.Yva8Kfp'] = new \Symfony\Component\DependencyInjection\Argument\ServiceLocator($this->getService, [
    'cmpt' => ['privates', '.errored..service_locator.Yva8Kfp.App\\Entity\\Compte', NULL, 'Cannot autowire service ".service_locator.Yva8Kfp": it references class "App\\Entity\\Compte" but no such service exists.'],
    'part' => ['privates', '.errored..service_locator.Yva8Kfp.App\\Entity\\Partenaire', NULL, 'Cannot autowire service ".service_locator.Yva8Kfp": it references class "App\\Entity\\Partenaire" but no such service exists.'],
    'validator' => ['services', 'validator', 'getValidatorService', false],
    'violations' => ['privates', '.errored..service_locator.Yva8Kfp.Symfony\\Component\\Validator\\ConstraintViolationList', NULL, 'Cannot autowire service ".service_locator.Yva8Kfp": it references class "Symfony\\Component\\Validator\\ConstraintViolationList" but no such service exists.'],
], [
    'cmpt' => 'App\\Entity\\Compte',
    'part' => 'App\\Entity\\Partenaire',
    'validator' => '?',
    'violations' => 'Symfony\\Component\\Validator\\ConstraintViolationList',
]);
